tests/test_resource.py

```diff
@@ -414,20 +421,54 @@ def test_get_pagination_multifield_query_params(self):
             },
         }
 
-    def test_no_prev_link_when_offset_minus_limit_lt_zero(self):
+    def test_no_prev_link_when_offset_minus_limit_lt_zero(self) -> None:
         links = self.resource.get_pagination(total_items=100, offset=10, limit=20)["links"]
         assert "prev" not in links
 
-    def test_no_next_returned_when_offset_plus_limit_gt_total_items(self):
+    def test_no_next_returned_when_offset_plus_limit_gt_total_items(self) -> None:
         links = self.resource.get_pagination(total_items=25, offset=20, limit=10)["links"]
         assert "next" not in links
 
-    def test_pagination_uri_with_existing_pagination_query_params(self):
+    def test_pagination_uri_with_existing_pagination_query_params(self) -> None:
         self.resource.request.query_params = MultiDict({"offset": "3", "limit": "42"})
         expected = "/test/path?offset={offset}&limit={limit}"
         assert self.resource._pagination_uri == expected  # pylint: disable=protected-access
 
-    def test_pagination_uri_without_query_params(self):
+    def test_pagination_uri_without_query_params(self) -> None:
         self.resource.request.query_params = MultiDict({})
         expected = "/test/path?offset={offset}&limit={limit}"
         assert self.resource._pagination_uri == expected  # pylint: disable=protected-access
+
+
+class TestEventAwareResource:
+    def test___init__(self) -> None:
```

Comment:
You did exactly the same thing as with the [\_\_call\_\_()](https://github.com/pdyba/lambdalizator/pull/50#discussion_r850446107) before. The name of this test brings almost no value when you don't mention what you expect.